src/Raven.Server/Documents/Handlers/Debugging/IdentityDebugHandler.cs

```diff
@@ -16,7 +17,7 @@ public async Task GetIdentities()
             using (Database.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             using (context.OpenReadTransaction())
             {
-                await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
+                await using (var writer = new AsyncBlittableJsonTextWriterForDebug(HttpContext.Request, context, ServerStore, ResponseBodyStream()))
```

Comment:
Let's revert this back to `AsyncBlittableJsonTextWriter`.

Reply:
done